openmeter/billing/provider/config.go

```diff
@@ -0,0 +1,63 @@
+package provider
+
+import "fmt"
+
+// Type specifies the provider used for billing
+type Type string
+
+const (
+	// TypeOpenMeter specifies the OpenMeter billing provider, which is a dummy billing provider mostly useful for testing and
+	// initial OpenMeter assessment
+	TypeOpenMeter Type = "openmeter"
+	// TypeStripe specifies the Stripe billing provider, which is a real billing provider that can be used in production
```

Comment:
This is an invoice provider, no? Not billing

Reply:
The openmeter will be a billing provider that is dummy (e.g. simulates all the functions including payments etc). Right now we don't have the distinction between tax, payments and invoicing providers, and I am reluctant to include that abstraction until we have at least one such use-case.